Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:12
```

Comment:
Maybe bump this to a more recent version? Looking at their website I think 14 or 15 is the latest stable version

Reply:
Done 🌮